<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teamLogin</class>
 <widget class="QWidget" name="teamLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="loginTab">
      <attribute name="title">
       <string>Login</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="loginButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="labelUsername">
           <property name="text">
            <string>Username :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditUsername"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelPassword">
           <property name="text">
            <string>Password :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEditPassword">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBoxSport">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelSport">
           <property name="text">
            <string> Sport :   </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mainTab">
      <attribute name="title">
       <string>Main Page</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="teamPanelLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;&lt;br/&gt;FUCK&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editTeamPushButton">
           <property name="text">
            <string>Edit Team</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="allTeamsPushButton">
           <property name="text">
            <string>All Current Teams</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="liveToursPushButton">
           <property name="text">
            <string>Ongoing Tournaments and Signup in them</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="incomingMatchesPushButton">
           <property name="text">
            <string>Incoming Matches</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="allAvailableToursPushButton">
           <property name="text">
            <string>All Available Tournaments</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
